api/inc/page_allocator_exports.h

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright (c) 2016, ARM Limited, All Rights Reserved
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#ifndef __UVISOR_API_PAGE_ALLOCATOR_EXPORTS_H__
+#define __UVISOR_API_PAGE_ALLOCATOR_EXPORTS_H__
+
+#include <stdint.h>
+#include <stddef.h>
+
+/* Must be a power of 2 for MPU alignment in ARMv7-M */
+#ifndef UVISOR_PAGE_SIZE
```

Comment:
Could you please add a comment on how to modify this from the user/developer perspective? There's still the old question of this symbol being available to both worlds, in- and out-of-uvisor, which might create conflicts.